docs/src/api/class-screenshotassertions.md

```diff
@@ -0,0 +1,41 @@
+# class: ScreenshotAssertions
+* langs: js
+
+Playwright provides methods for comapring page and element screenshots with
+expected values stored in files. See also [`method: PageAssertions.toHaveScreenshot`] and
+[`LocatorAssertions.toHaveScreenshot`].
+
+```js
+expect(screenshot).toMatchSnapshot('landing-page.png');
+```
+
+<!-- TOC -->
+
+## method: ScreenshotAssertions.toMatchSnapshot
```

Comment:
I'd keep `toMatchSnapshot` inside test-assertions-js.md, since this gets never added to the language bindings. Then we don't need the extra class.

It's successor is toHaveScreenshot.

Reply:
They are different things - one is waiting, the other is not